db/src/main/java/org/odk/collect/db/sqlite/SQLiteDatabaseExt.kt

```diff
@@ -35,4 +35,27 @@ object SQLiteDatabaseExt {
 
         return columnNames.toList()
     }
+
+    @JvmStatic
+    @JvmOverloads
+    fun SQLiteDatabase.addColumn(
```

Comment:
This could be simplified:
```
@JvmStatic
@JvmOverloads
fun SQLiteDatabase.addColumn(
    table: String,
    column: String,
    type: String,
    default: String? = null
) {
    val defaultClause = default?.let { " DEFAULT $it" } ?: ""
    execSQL("ALTER TABLE $table ADD $column $type$defaultClause;")
}
```

Reply:
I actually find that harder to follow honestly. Are you ok for us to keep it how it is?

Reply:
ok.